.github/ISSUE_TEMPLATE/user-story-issue.md

```diff
@@ -0,0 +1,25 @@
+---
+name: User Story
+about: This template provides a basic structure for user story issues.
+labels: Needs Triage
+---
+
+# Description
+As a... <persona>, I want to... <user action>, so I can... <user value>
+
+<Link to Design if available>
+
+# Acceptance criteria
+
+- [ ] This is something that can be verified to show that this user story is satisfied.
+- [ ] Unit Tests are completed and code passes tests
+
+# Dependencies
+
+
+# Sprint-Ready Checklist 
+- [ ] Acceptance criteria defined 
+- [ ] Acceptance criteria is verifiable / testable 
+- [ ] External / 3rd Party dependencies identified
+- [ ] Engineering team understands acceptance criteria 
```

Comment:
Still don't love the `understands` part of this, but if you decide to keep it as the key word in this readiness check, it would be good for us to have a conversation as a team about how we will know whether or not we have reached said understanding.  (Such a definition is probably better suited for a working agreement, rather than this template though, so no changes necessary here)